src/OrbitGl/TimeGraph.cpp

```diff
@@ -57,15 +57,18 @@ using orbit_gl::VariableTrack;
 
 TimeGraph::TimeGraph(AccessibleInterfaceProvider* parent, OrbitApp* app,
                      orbit_gl::Viewport* viewport, CaptureData* capture_data,
-                     PickingManager* picking_manager, TimeGraphLayout* time_graph_layout)
+                     PickingManager* picking_manager,
+                     orbit_gl::BatchRenderGroupStateManager* render_group_manager,
+                     TimeGraphLayout* time_graph_layout)
     // Note that `GlCanvas` and `TimeGraph` span the bridge to OpenGl content, and `TimeGraph`'s
     // parent needs special handling for accessibility. Thus, we use `nullptr` here and we save the
     // parent in accessible_parent_ which doesn't need to be a CaptureViewElement.
     : orbit_gl::CaptureViewElement(nullptr, viewport, time_graph_layout),
       accessible_parent_{parent},
+      text_renderer_static_(),
```

Comment:
Same as previously. Not needed anymore, right?

Reply:
Indeed, thanks.